vercel/turborepo
# chore(linting): implement ESLint flat config compatibility for `eslint-plugin-turbo`

### Description

- Add "flat/recommended" config to `eslint-plugin-turbo`
- Add `meta` property to plugin
- Update README.md accordingly with information on using flat config
- Add tests for flat config in `__tests__/cwdFlat.test.ts`

### Testing Instructions

Run the following command:

```shell
pnpm run test --filter=eslint-plugin-turbo
```

See vercel/turborepo#8606
